openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -109,6 +114,33 @@
                             }
                         );
                     }
+
+                    $manageLearnersTab.find('.' + view.SLIDABLE_CLASS).click(
+                        function(event) {
+                            var $toggle = $(event.currentTarget),
+                                $content = $toggle.next('.' + view.SLIDABLE_CONTENT_CLASS);
+
+                            if ($toggle.hasClass(view.IS_SHOWING_CLASS)) {
+                                $toggle.removeClass(view.IS_SHOWING_CLASS).attr('aria-expanded', 'false');
```

Comment:
Note to reviewers: the "is--showing" class is necessary for the collapse/expand icon to update.